docs/access/ssh.md

```diff
@@ -174,6 +174,53 @@ ssh -A cscsusername@ela.cscs.ch
 ssh daint.cscs.ch
 ```
 
+## SSH tunnel to a service on CSCS Alps (e.g., daint) via ela
+
+If you have a server listening on a compute node in an Alps cluster and want to reach it from your laptop. Here is a quick solution: allocate a node, start your server bound to `localhost`, open an SSH tunnel that jumps through `ela` to the cluster (e.g. `daint`), then use `http://localhost:PORT` on your laptop.
+This is how:
+
+### Requirements
```

Comment:
I find the subheadings add little value (especially when the sections are so short), so I'm going assume they get removed in the following comments, but if you think they are useful let me know.